docs/content/policy-reference.md

```diff
@@ -266,6 +266,32 @@ f(x) = "B" { x >= 80; x < 90 }
 f(x) = "C" { x >= 70; x < 80 }
 ```
 
+### Reference Heads
+
+```live:rules/ref_heads:module:read_only
+fruit.apple.seeds = 12 { input == "apple" } # complete document
+
+fruit.orange.color(x) = true { x == "orange" } # function
+
+fruit.citron.colors[x] { x := "yellow" } # partial set
+
+fruit.banana.phone[x] = "bananular" { x := "cellular" } # partial object
+```
+
+For reasons of backwards-compatibility, partial sets containing _strings_ need
+to use variables, i.e.
+
+```live:rules/ref_heads/ps_complete:module:read_only
+fruit.box["apples"] { true }
+```
+
+defines a _complete document rule_ `fruit.box.apples` with value `true`. To define
+a partial set rule with a reference head, use
+
+```live:rules/ref_heads/ps_set_of_strings:module:read_only
+fruit.box[x] { x := "apples" }
+```
```

Comment:
I hadn't thought about the ambiguity between partial sets and the new syntax. I think we need to come up with a different solution to this because I would not want to preclude us from supporting variables in the refs in the future... e.g., `foo.bar[x].baz[y] { ... }` should define a nested object structure:

```
foo = {
  "bar": {x0: {"baz": {y0: true, y1: true, ..., yN: true}, x1: {...}, ..., xM: {...}}}
}
```